core/sdk/build.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Comcast Cable Communications Management, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+extern crate vergen;
+use vergen::{vergen, ConstantsFlags};
+
+fn main() {
```

Comment:
SDK should not need a main

Reply:
This main for build, not main (main) (build.rs, not main.rs). This is required to get the version to be generated correctly.